### DISCUSSION TITLE
Rust support for edge functions

### DISCUSSION THREAD

**Discussion by lostb1t** [supabase/supabase#18809] [Feature Requests]
Currently typescript and wasm are supported. Are there any plans to add first class support for Rust? either native or through wasm? much like https://blog.cloudflare.com/workers-rust-sdk/

**Comment by ilbertt** [comment]
I'd really like to write Edge Functions in Rust! Maybe [Deno's wasmbuild](https://deno.com/blog/wasmbuild) can help?
